### PR TITLE
Update with 2021 roadmap items

### PR DESCRIPTION
**Pull request by thisisdano**
[Preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/dw-update-2021-roadmap/about/product-roadmap/) →

### CODE REVIEW

File: pages/whats-new/product-roadmap.md [hunk 1]
```diff
@@ -16,10 +16,54 @@ Below is our product roadmap: a long-term plan of the goals, features, and long-
 <!-- TODO: Make these into issues; add roadmap project board
 You can also <a href="https://github.com/uswds/uswds/milestone/52" class="">view our product roadmap goals on GitHub</a>. -->
 
-<h2>2020 Roadmap</h2>
+<h2>Product Roadmap</h2>
 
 <div class="maxw-tablet">
   <ul class="site-roadmap-list">
+    <li class="site-roadmap-list__item"><span class="flex-fill">Value and benefits</span>
```
Comment (mejiaj) [inline pages/whats-new/product-roadmap.md:23]: We should change these headers to `flex-auto` or `flex-column`. IE11 seems to have trouble with it:

image

**Review by afeijoo — APPROVED** [review]
List components A-Z

File: pages/whats-new/product-roadmap.md (hunk 1 above)
Comment (mejiaj) [reply pages/whats-new/product-roadmap.md]: Resolved in 0679bab

**Review by mejiaj — APPROVED** [review]

**Comment by thisisdano** [comment]
Thanks!

**Pull request merged**